cmd/config_validation_test.go

```diff
@@ -402,3 +407,86 @@ func TestValidateConfigFile_GCSConnectionConfigSuccessful(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateConfigFile_FileSystemConfigSuccessful(t *testing.T) {
+	defaultDirMode, err := strconv.ParseInt("0755", 8, 0)
+	require.NoError(t, err)
+	defaultFileMode, err := strconv.ParseInt("0644", 8, 0)
+	require.NoError(t, err)
+	fileMode666, err := strconv.ParseInt("0666", 8, 0)
+	require.NoError(t, err)
+	dirMode777, err := strconv.ParseInt("0777", 8, 0)
+	require.NoError(t, err)
+	hd, err := os.UserHomeDir()
+	require.NoError(t, err)
+	testCases := []struct {
+		name           string
+		configFile     string
+		expectedConfig *cfg.Config
+	}{
+		{
+			name:       "Empty config file [default values].",
+			configFile: "testdata/empty_file.yaml",
+			expectedConfig: &cfg.Config{
+				FileSystem: cfg.FileSystemConfig{
+					DirMode:                cfg.Octal(defaultDirMode),
```

Comment:
Ditto for other cases
```suggestion
					DirMode:               0755,
```